spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
 require 'rails_helper'
 
 RSpec.describe V0::SessionsController, type: :controller do
-  let(:mvi_user) { FactoryGirl.build(:mvi_user) }
+  let(:mvi_user) { FactoryGirl.build(:loa3_user) }
```

Comment:
@omgitsbillryan is this correct? Sessions controller still has a couple broken specs wasn't sure in what state the user should be here.